### ISSUE TITLE
Feature: Delegate revisit errors to query siblings

### ISSUE THREAD

**Issue by Avokadoen** [nlnwa/gowarcserver#45]
**Is your feature request related to a problem? Please describe.**
As gowarcserver becomes more used with bigger datasets the need for splitting bigger collections might occur.

**Describe the solution you'd like**
Files can inherently be split since warc records are a thing.  Add support for config files to express indexing a subsection of a file. 

Example: 

config for instance 1, index and "own" records in warc including record 0 to 10 
```yaml
indexFiles:
    - example.warc:0-10
```

config for instance 2, index and "own" records in warc including record 11 and the remaining in the file
```yaml
indexFiles:
    - example.warc:11- 
```
The configuration should also be set up in a way so that instance 1 and 2 are sibilings (common parent) in order for queries about example.warc to work.

**Describe alternatives you've considered**
None

**Additional context**
None

**Comment by Avokadoen** [comment]
[content handler](https://github.com/nlnwa/gowarcserver/blob/master/pkg/server/contenthandler.go) should be useful here ...

**Comment by Avokadoen** [comment]
Issue description is somewhat wrong, but this is resolved in #57
